VKClone/ViewControllers/LoginViewController.swift

```diff
@@ -0,0 +1,171 @@
+// LoginViewController.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+/// LoginViewController экран логина
```

Comment:
```suggestion
///  экран логина
```
LoginViewController названия контролера не пиши больше пожалуйста